css/styles.css

```diff
@@ -324,3 +324,106 @@ Colors:
   border: 0.5rem solid rgb(255 255 255 / 9%);
   border-radius: 50%;
 }
+
+/* ***************** */
+/* Menu */
+/* ***************** */
+.menu {
+  padding: 10.4rem 27.4rem 9.6rem;
+  background-image: linear-gradient(
+      to right bottom,
+      rgb(32 90 91 / 14%),
+      rgb(255 255 255)
+    ),
+    url("../img/bg-menu.png");
+  box-shadow: inset 0 0 2.4rem 1.2rem rgb(0 0 0 / 6%);
+}
+
+.menu-content {
+  display: flex;
+  flex-wrap: wrap;
+  column-gap: 14.6rem;
```

Comment:
Why not just `gap`? `column-gap` belongs to multi-column layout.

Reply:
This was before I made two columns, and then forgot to redo.